pkg/services/updates.go

```
@@ -533,3 +534,19 @@ func (s *UpdateService) UpdateDevicesFromUpdateTransaction(update models.UpdateT
 
 	return nil
 }
+
+// ValidateUpdateSelection validate the images for update
+func (s *UpdateService) ValidateUpdateSelection(account string, imageIds []uint) (bool, error) {
+	var count int64
+
```

let's return an error when len(imagesIds) == 0

---

Added on the route function